### DISCUSSION TITLE
Strange download error 406

### DISCUSSION THREAD

**Discussion by Vzz1c** [ScoopInstaller/Scoop#5188] [Help and Questions] [answered]
{
    "version": "2.9.3.3",
    "description": "Systemlink",
    "homepage": "https://schinagl.priv.at/nt/ln/ln.html",
    "license": "Proprietary",
    "url": "https://schinagl.priv.at/nt/ln/ln64.zip",
    "hash": "27fadef5143aa76e3840b96f2626bb9d6d511bdd889979513d62d3994bde39d2",
    "bin": "ln.exe",
    "checkver": {
        "url": "https://schinagl.priv.at/nt/ln/ln.html",
        "regex": "Last\\sUpdated[\\s\\S]*?Version\\s(\\S+)<\\/font>"
    },
    "autoupdate": {
        "url": "https://schinagl.priv.at/nt/ln/ln64.zip"
    }
}

=================================================================
This list works fine for automatic update detection.
.\bin\checkver.ps1 runs without problems.
The links are also fine.
The problem is that it cannot be installed or downloaded
scoop install and scoop download
It shows network error 406
I tried to use aria2, and set up a proxy
Again, I can't download this link:
https://schinagl.priv.at/nt/ln/ln64.zip
But I was able to download it using the wget command, which has nothing to do with proxies
wget https://schinagl.priv.at/nt/ln/ln64.zip -o Test.zip
===================================
So, what's the problem, it's a simple setup, but I don't know what the problem is

**Comment by rashil2000** [comment, marked as answer]
Something related to your network. 


![image](https://user-images.githubusercontent.com/46838874/195819012-2a133a21-c6e9-4250-baa1-0a32c5f371be.png)


Try using `irm https://schinagl.priv.at/nt/ln/ln64.zip -outfile ln64.zip`

**Reply by Vzz1c** [reply]
![image](https://user-images.githubusercontent.com/49944812/195826020-dba87738-de83-436a-827a-7f5eae7b6385.png)
Everything else is normal, only this one cannot be downloaded, which is very strange..

**Reply by Vzz1c** [reply]
irm or wget or curl can link this file normally, there should be no problem with my network

**Reply by Vzz1c** [reply]
The same LAN, another computer can be normal, this is really a problem of my network, I can not know why, because everything else is always normal, this is too strange, I think I should find time to reinstall the system ...

**Reply by Vzz1c** [reply]
I suddenly found out that the problem is the program, it turns out that I have been using Ash258/Scoop-Core, which is amazing, I just replaced it with ScoopInstaller/Scoop, I was wondering why the debug function does not work, and sometimes there are some strange problems, but now everything is fine... Thank you!

**Reply by rashil2000** [reply]
> it turns out that I have been using Ash258/Scoop-Core

This is why we ask people to fill out the issue template (which you had ignored), which contains these details. The problem would have been recognized right away.
